packages/patrol_gen/lib/src/generators/dart/dart_contracts_generator.dart

```diff
@@ -15,7 +15,9 @@ class DartContractsGenerator {
       buffer.writeln(_createMessage(messageDefintion));
     }
 
-    final content = DartFormatter().format(buffer.toString());
+    final content = DartFormatter(
+      languageVersion: DartFormatter.latestLanguageVersion,
```

Comment:
Need confirmation from mobile positive person that this is fine

Reply:
Ok I see now that we've introduced `dart format "$patrol_dart_out_dir"` line to `./gen_from_schema` some time ago so my final answer is:
It doesn't matter what we use here because the output will be formatted inside `package/patrol` anyway